CONTRIBUTING.md

```diff
@@ -1,41 +1,55 @@
 # How To Contribute
-Contributing to Gaia Dimension has various ways to contribute to the development. From a standpoint, contributions can include Coding, Assets and Testing. Please note that any PRs made can and will be rejected if they do not comply with the following guidelines. PRs will be closed if:
+There are various ways to contribute to the development of the Gaia Dimension mod. Some common types of contributions can include donating code, making textures, translating, and testing. Please note that any Pull Requests (PRs) made can and will be rejected if they do not comply with the following guidelines.
 
-* Does not comply with the listed guielines
-* No action is done to resolve conflicts or build failures within 2 weeks
-
-## Coding
-Though it is not easy from a surface level, it's worth more to contribute via coding than any other place. However, there are a few things to note when submitting a PR for coding.
+## Code
+Though it is not easy, donating code is the best way to contribute. However, there are a few things to note when submitting a PR with code.
 
 ### Make sure braces are on *the same line*
 As much as some people are inclined to, and for consistency's sake, make sure your code is formatted to use same-line braces. This is to reduce on file size and better vertical readability.
 
 Example:
-```
+```java
 public void() {
 }
 ```
+### Spacing must use four spaces, not tabs
```

Comment:
Just need an extra space between the code block and the header here, just for cleanliness.